### ISSUE TITLE
Docs for downloading new dictionaries or updating downloaded dictionaries

### ISSUE THREAD

**Issue by drdhaval2785** [hrishikeshrt/PyCDSL#4]
* Python Interface to Cologne Digital Sanskrit Lexicon (CDSL) version: 0.1.9
* Python version: 3.6.9
* Operating System: Bodhi Linux 9

### Description

Currently the docs are missing on this poing.
I am seeing 4 dictionaries downloaded.
Whereas the message you have copy pasted on the docs have 23 dictionaries.
It should be documented how the user can download the dictionaries of his choice or update the dictionaries.

### What I Did

```
$ cdsl
Cologne Sanskrit Digital Lexicon (CDSL)
Type any keyword to search in the selected lexicon. (help or ? for list of options)
Loaded 4 dictionaries.

(CDSL::None) 
```

**Comment by hrishikeshrt** [comment]
Typing `help` or `?` shows the other commands.

User can see available dictionaries using `available` command, and use any dictionary using `use DICT_ID` syntax. If the dictionary is not available, it gets downloaded and loaded.

I will mark this as TODO for documentation update.

Thank you.

**Comment by hrishikeshrt** [comment]
Fixed in 99659842f02a9e5d6deebe5d344f6bd33180aaa8.

**Comment by drdhaval2785** [comment]
great
